lib/core/utils/responsive.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+
+import 'constant.dart';
+
+class Responsive {
+  late double? blockSizeHorizontal;
+  late double? blockSizeVertical;
+  late double? textRatio;
```

Comment:
i don't think we need to make it an optional after adding `late` keyword

Reply:
done